src/main/groovy/wooga/gradle/version/VersionPluginExtension.groovy

```diff
@@ -309,6 +315,10 @@ trait VersionPluginExtension implements BaseSpec {
         releaseStage
     }
 
+    Provider<GitVersionRepository> versionRepo = git.map {
+        Grgit it -> GitVersionRepository.fromTagStrategy(it, new PrefixTagStrategy(prefix.get(), true))
```

Comment:
can we also please use the same style for closueres etc. The parameters `Grgit it ->` should be on the line before if you need to split the closure in multiple lines.